README.md

```diff
@@ -60,4 +78,6 @@ Now you should be able to query
 | default        |
 +----------------+
 1 row selected (1.921 seconds)
-```
\ No newline at end of file
+```
+
+You can access YARN here http://localhost:8088/cluster
```

Comment:
@alex-ber I would add `-p 8088:8088` to the `docker run` command.

Reply:
8088 is part of the range 8030-8088